src/updateFunctions.test.js

```diff
@@ -0,0 +1,561 @@
+import { updateData, addMovement, findDeleteOrDecrease, updateCurrAcc, updateAccsLend } from './updateFunctions';
+import { accounts, date } from './data';
+import { updateAccsRepay } from './updateAccsRepay';
+
+/// 24.02.2021 Working on tests
+////////// Testing updateAccsRepay()
+
+describe("updateAccsRepay()", () => {
+    const acc1 = {
+        owner: 'Acc1',
```

Comment:
Tip of the day: If you refactor your tests, and move test data generation into functions, you can make your test files shorter. Happy to show you what I mean later.

Reply:
I am really looking forward to see it 👯